tests/phpunit/WC_Stripe_Payment_Gateway_Test.php

```
@@ -844,4 +845,148 @@ public function test_invalid_request_error_with_non_blocked_code_is_retryable()
+				'response' => 200,
+				'headers'  => [ 'Content-Type' => 'application/json' ],
+				'body'     => json_encode( $mandate ),
+			];
+		};
+
+		add_filter( 'pre_http_request', $mock_request, 10, 3 );
+
+		$actual = $this->gateway->disable_subscription_edit_for_india( true, $order );
+
+		// Clean up.
+		\WC_Stripe_Database_Cache::delete( 'mandate_for_subscription_' . $order->get_id() );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\WC_Stripe_Database_Cache' in method 'test_disable_subscription_edit_for_india'. (undefined)

(StaticAccess)